exampleSite/content/fragments/repositories/index.md

```diff
@@ -0,0 +1,18 @@
++++
+fragment = "content"
+weight = 100
+
+title = "Repositories"
+background = "light"
++++
+
+Display user repositories from Github.
+
+<!--more-->
+
+<div class="alert alert-primary" role="alert">
+{{% md %}}
+For now, we only support Github. You can review the [documentation for their API](https://docs.github.com/en/rest/reference/repos#list-repositories-for-a-user).  
+We will add support for more sevices in the future.
```

Comment:
Not with the current configuration adding more services would be possible. It feels quite focussed on github. Adding other services would need more data from the user such as sub items with link to project on say gitlab or github.

We could leave it this simple and make it a first provider fragment: "repositories_github" for example.

Reply:
So rename the fragment to repositories_github and remove the alert?

I was thinking about adding a `provider` and `framework` (or `variation`) variables instead of naming convention. It's not related to this issue though, just wanted to mention it somewhere since I tend to forget a lot :D